meta-mel/mel-iot/recipes-iot/libcoap/libcoap_%.bbappend

```diff
@@ -0,0 +1,2 @@
+EXTRA_OECONF_append_mel = " --datadir=/opt"
```

Comment:
This line is pointless, for example. It's also just wrong, since it could easily affect more than just examples/samples. But if you're going to take this approach, then just drop `EXTRA_OECONF` and do this:
```
datadir = "/opt"
```

As `--datadir=${datadir}` is already passed to ./configure in autotools.bbclass.